docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -220,12 +221,15 @@ Json Parameters:
       container
 -   **ExposedPorts** - An object mapping ports to an empty object in the form of:
       `"ExposedPorts": { "<port>/<tcp|udp>: {}" }`
+-   **VolumeDriver** - A string value containing the volume driver to use, `local` by default.
```

Comment:
It would be good to add to this sentence to define the relationship between drivers and plguins:

    Volume drivers are either the string "local" to use the built-in local volume
    driver, or the name of a [volume plugin](/userguide/plugins) to use a volume
    plugin as a driver.